<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>yoDownet</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="urlView">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>64</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="addAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="pauseAction"/>
    <addaction name="resumeAction"/>
    <addaction name="stopAction"/>
    <addaction name="removeAction"/>
    <addaction name="removeFromListAction"/>
   </widget>
   <widget class="QMenu" name="settingsMenu">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="showMenubarAction"/>
    <addaction name="showToolbarAction"/>
    <addaction name="showStatusbarAction"/>
    <addaction name="separator"/>
    <addaction name="preferencesAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="reportBugAction"/>
    <addaction name="aboutQtAction"/>
    <addaction name="aboutyoDownetAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="settingsMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="addAction"/>
   <addaction name="pauseAction"/>
   <addaction name="resumeAction"/>
   <addaction name="stopAction"/>
   <addaction name="removeFromListAction"/>
   <addaction name="separator"/>
   <addaction name="exitAction"/>
  </widget>
  <action name="addAction">
   <property name="icon">
    <iconset theme="list-add">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Add</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="pauseAction">
   <property name="icon">
    <iconset theme="media-playback-pause">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="resumeAction">
   <property name="icon">
    <iconset theme="media-playlist-repeat">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Resume</string>
   </property>
  </action>
  <action name="removeAction">
   <property name="icon">
    <iconset theme="edit-delete">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="removeFromListAction">
   <property name="icon">
    <iconset theme="list-remove">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Remove From List</string>
   </property>
  </action>
  <action name="reportBugAction">
   <property name="text">
    <string>Report Bug...</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="aboutyoDownetAction">
   <property name="icon">
    <iconset resource="../resource/icons.qrc">
     <normaloff>:/icons/images/yoDownet_64.svg</normaloff>:/icons/images/yoDownet_64.svg</iconset>
   </property>
   <property name="text">
    <string>About yo&amp;Downet</string>
   </property>
  </action>
  <action name="showMenubarAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Menubar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="showToolbarAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Toolbar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="showStatusbarAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Statusbar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="preferencesAction">
   <property name="icon">
    <iconset theme="preferences-system">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="stopAction">
   <property name="icon">
    <iconset theme="media-playback-stop">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resource/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exitAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showStatusbarAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>statusbar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>588</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showToolbarAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolbar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showMenubarAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>menubar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
